stylesheets/_layout.tabs.scss

```diff
@@ -232,3 +232,7 @@
         @include col-span-width(9);
     }
 }
+
+.tab__header {
+    width: 100%;
```

Comment:
I'd move this up with flex-basis as we know it's always going to be a div and I suspect that the width is only needed when flexing?

Unless there's another reason why we need to specify it?